book/src/integrations/oauth2.md

```diff
@@ -459,57 +459,78 @@ Each client has unique signing keys and access secrets, so this is limited to ea
 ## WebFinger
 
 [WebFinger](https://datatracker.ietf.org/doc/html/rfc7033) provides a mechanism
-for discovering information about people or other entities. It can be used by an
-identity provider to supply OpenID Connect discovery information.
+for discovering information about entities at a well-known URL
+(`https://{hostname}/.well-known/webfinger`).
 
-Kanidm provides
-[an Identity Provider Discovery for OIDC URL](https://datatracker.ietf.org/doc/html/rfc7033#section-3.1)
-response to all incoming WebFinger requests, using a user's SPN as their account
-ID. This does not match on email addresses as they are not guaranteed to be
-unique.
+It can be used by a WebFinger client to
+[discover the OIDC issuer URL](https://datatracker.ietf.org/doc/html/rfc7033#section-3.1)
+of an identity provider from the hostname alone, and seems to be intended to
+support dynamic client registration flows for large public identity providers.
 
-However, WebFinger has a number of flaws which make it difficult to use with
-Kanidm:
+Kanidm v1.5.1 and later can respond to WebFinger requests, using a user's SPN as
```

Comment:
I preferred the version in a little note at the top so admins don't need to read the entire text to know what version they need to update to

Reply:
@yaleman any opinions on this? 😅

Reply:
It's 1.5.0, but also I think we don't need it because docs are published per-server-version, so when we merge this it won't show until 1.6 iirc.